### PR TITLE
feat(TMRX-1444): Update image on mobile web for Lead 1 and Lead 3

### PR DESCRIPTION
**Pull request by ibrahim-kurhan**
### Description

Himesh has requested that we show the image in mobile web for the first article in the lead 1 & 3 2 ices. 

Designs here: 

Lead 1 https://www.figma.com/file/C1K0ePqSX0irobX6lAMq7h/Sport-section?type=design&node-id=274%3A129455&mode=design&t=o5auQUx6vwJntLTS-1 - Connect your Figma account

Lead 3 https://www.figma.com/file/C1K0ePqSX0irobX6lAMq7h/Sport-section?type=design&node-id=274%3A178284&mode=design&t=o5auQUx6vwJntLTS-1 - Connect your Figma account 
[JIRA-1234](https://nidigitalsolutions.jira.com/browse/JIRA-1234)

### Checklist

- [x] Have you done any manual testing?
- [ ] Does it have automated tests?
- [ ] Do you need any other PRs merged before this (if so please list)?
- [ ] Do you need to update the README/Runbook
- [ ] Have you checked for [accessibility](https://nidigitalsolutions.jira.com/wiki/spaces/TNLDigital/pages/3898048523/Accessibility+Dev+Guide#Checklist)?

### Screenshots (if appropriate):
Lead 1
Screenshot 2023-08-10 at 15 25 20
Lead 3
Screenshot 2023-08-10 at 15 25 58

### CODE REVIEW

File: packages/ts-newskit/package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
     "fix:prettier": "prettier \"src/**/*.{ts,tsx}\" \"__tests__/**/*.tsx\" --write",
     "fix:tslint": "tslint --fix --project .",
     "lint": "tslint --project . && prettier \"src/**/*.{ts,tsx}\" --list-different",
+    "prepublishOnly": "yarn transpile && yarn bundle",
```
Comment (nishprogrammes) [inline packages/ts-newskit/package.json:30]: Was this addition intended with this PR?

**Review by nishprogrammes — APPROVED** [review]

**Review by douglasmik — APPROVED** [review]

**Review by adamosborne-tnl — APPROVED** [review]

**Review by bibicollins1 — APPROVED** [review]
